test/journeys/specs/multiple/index.js

```diff
@@ -7,6 +7,7 @@ import CiscoSpark from '@ciscospark/spark-core';
 import testUsers from '@ciscospark/test-helper-test-users';
 
 import waitForPromise from '../../lib/wait-for-promise';
+import {moveMouse} from '../../lib/test-helpers/index';
```

Comment:
I would suggest one of two options here:
- use `import {moveMouse} from '../../lib/test-helpers';`
- rename to `mouse.js` and do `import {moveMouse} from '../../lib/test-helpers/mouse;`

There's no need to specify `index`

Reply:
oh, good point, i'll remove the `index`